cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -2,21 +2,24 @@
 
 {% macro render(value, settings, form_id) %}
 
-    {% set form = settings.form %}
-
-    {# JS-disabled result handling #}
-
-<div class="o-feedback
```

Comment:
@virginiacc Shoot, I didn't realize that. Is there any chance we can change to either a data attribute or a `js-` prefixed class so this doesn't happen again in the future?